iron-input.html

```diff
@@ -68,18 +68,20 @@
       },
 
       /**
-       * Set to true to prevent the user from entering invalid input. The new input characters are
-       * matched with `allowedPattern` if it is set, otherwise it will use the `type` attribute (only
-       * supported for `type=number`).
+       * Set to true to prevent the user from entering invalid input. If `allowedPattern` is set, 
+       * The new input characters are matched with that. If more than one character is typed or pasted in,
```

Comment:
Oh, perfect! Sorry, I had misunderstood from your original issue description that paste was borked all around. If it rejects all characters, then that's 👌
Feel free to rejiggle my original suggestion to include this :)